extensions/rich_text_components/Math/directives/oppia-noninteractive-math.directive.ts

```diff
@@ -41,6 +49,52 @@ angular.module('oppia').directive('oppiaNoninteractiveMath', [
             $attrs.mathContentWithValue);
           if (mathExpressionContent.hasOwnProperty('raw_latex')) {
             ctrl.rawLatex = mathExpressionContent.raw_latex;
```

Comment:
Do we still need to bind this value to the ctrl?

Reply:
not needed, removed it.